packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/CurrentProduct.tsx

```diff
@@ -1,140 +1,121 @@
+interface ISubscriptionWidget {
+  className?: string
+}
+
+const CurrentProduct = ({ className }: ISubscriptionWidget) => {
   const classes = useStyles()
   const { storageSummary } = useFiles()
+  const { currentSubscription } = useBilling()
 
-  return (
-    <Grid container>
-      <Grid
-        item
-        xs={12}
-        sm={12}
-        md={12}
-      >
-        {storageSummary &&
-        <div className={classes.container}>
-          <div className={classes.storageBox}>
-            <Typography
-              variant="h4"
-              component="h4"
-              className={clsx(classes.margins, classes.storagePlan)}
-            >
-              <Trans>Storage Plan</Trans>
-            </Typography>
-            <Typography
-              variant="h5"
-              component="h5"
-              className={clsx(classes.earlyAdopter)}
-            >
-              {t`Early Adopter: Free up to ${formatBytes(
-                storageSummary.total_storage, 2
-              )}`}
-            </Typography>
-            <Typography
-              variant="body1"
-              component="p"
-              className={clsx(classes.margins, classes.subtitle)}
-            >
-              {t`Your first ${formatBytes(
-                storageSummary.total_storage, 2
-              )} are free, and you’ll get a discount on our monthly plan once you need more than that`}.
-            </Typography>
-          </div>
-          <div className={classes.essentialContainer}>
-            <div className={classes.spaceUsedBox}>
-              <Typography
-                variant="body1"
-                className={classes.spaceUsedMargin}
-                component="p"
-              >{t`${formatBytes(storageSummary.used_storage, 2)} of ${formatBytes(
-                  storageSummary.total_storage, 2
-                )} used (${Math.ceil(storageSummary.used_storage / storageSummary.total_storage) * 100}%)`}
-              </Typography>
-              <ProgressBar
-                className={classes.spaceUsedMargin}
-                progress={(storageSummary.used_storage / storageSummary.total_storage) * 100}
-                size="small"
-              />
-            </div>
-            <Link
-              className={classes.link}
-              to={ROUTE_LINKS.Plans}
+  return (<section className={clsx(classes.root, className)}>
+    <Typography
+      variant="h4"
+      component="h2"
+    >
+      <Trans>Your plan</Trans>
+    </Typography>
+    {
+      currentSubscription
+        ?  <Typography
+          variant="h5"
+          component="h5"
+        >
+          {
+            currentSubscription?.product.name
+          }
+        </Typography>
+        : <Loading />
+    }
+    {storageSummary &&
+      <>
+        <div className={classes.spaceUsedBox}>
+          <Typography
+            variant="body2"
+            component="p"
+          >
+            {`${formatBytes(storageSummary.used_storage, 2)} of ${formatBytes(
```

Comment:
needs translation, not sure how, with `t` should work
```suggestion
            {t`${formatBytes(storageSummary.used_storage, 2)} of ${formatBytes(
```